### ISSUE TITLE
Problems updating tasks close to the end of the day

### ISSUE THREAD

**Issue by ErideDLF** [HabitRPG/habitica#974]
yesterday night I tried to log into the page but the server was having some troubles and I couldn't check the tasks I did. And this morning when i've logged in again I was dead.
It isn't a big problem because i've only dropped one level , and I know the system is under construction, but it isn't the first time it happens to me.Is anyone having the same problem?

**Comment by Longformer** [comment]
Yes ErideDLF, I've had the same problem.  On a related note I've had times where I've updated a few tasks before going to bed, Habit said it was connected but then the next morning I lost the HP for them anyway.  I've also been able to tell which tasks went back to being undone, because often I'm seeing yesterday's state (before marking those last few) when I look at it in the morning. I've checked my reset time and it's still at midnight.
